REFERENCE.md

```diff
@@ -213,6 +216,13 @@ The cross-validation procedure works as follows:
 
 Note that `honesty.fraction` and `prune.empty.leaves` are only considered for tuning when `honesty = TRUE` (its default value). Parameter tuning does not try different options of `honesty` itself.
 
+### Merging forests
+
+GRF allows users to grow their forests separately and then create a single forest using the function `merge_forests`.
+
+The main motivation for this functionality is that, in order to ensure valid predictions, we recommend that users grow as many trees as necessary to ensure that the values in `excess.error` are negligible relative to `variance.estimates`. However, it is often difficult to know how many trees are necessary for this condition to be met. In that case, users can iteratively grow forests and check if `excess.error` is small enough.
```

Comment:
We haven't actually explained what 'excess error' is, do you think it would be helpful to give a one-sentence definition here before referencing it?